### DISCUSSION TITLE
Error with babel plugin transform-react-jsx: pragmaFrag not set

### DISCUSSION THREAD

**Discussion by aegatlin** [vercel/next.js#11244] [Help] [answered]
```
transform-react-jsx: pragma has been set but pragmaFrag has not been set
```

I haven't ran into this before. I'm setting up a new project using next v 9.3.1, and I'm getting this error when trying to use React Fragments. It wasn't a problem on my last project in next v 9.1.1. Has something changed?

I have no `.babelrc` file. I'm using typescript v 3.8.3 and react v 16.13.1.

I can provide more context if yall think its an app setup issue. I just want to make sure this isn't "expected" or something.

**Comment by timneutkens** [comment, marked as answer]
Duplicate of https://github.com/zeit/next.js/issues/11230
